apps/src/templates/sectionAssessments/sectionAssessmentsRedux.js

```diff
@@ -73,7 +73,8 @@ export const asyncLoadAssessments = (sectionId, scriptId) => {
     const state = getState().sectionAssessments;
 
     // Don't load data if it's already stored in redux.
-    if (state.assessmentResponsesByScript[scriptId]) {
+    // Don't attempt to load data if there is not script or section selected.
+    if (state.assessmentResponsesByScript[scriptId] || !scriptId || !sectionId) {
```

Comment:
apologies if this is just a nit, but the comment says "if there is not a script OR section selected" but the code appears to implement "if there is not a script AND section selected". I'm not sure which is correct, but they should probably be consistent